app/src/test/java/org/simple/clinic/selectcountry/SelectCountryUpdateTest.kt

```diff
@@ -97,4 +97,18 @@ class SelectCountryUpdateTest {
             hasEffects(GoToNextScreen as SelectCountryEffect)
         ))
   }
+
+  @Test
+  fun `when retry is clicked, then fetch manifest`() {
+    val model = defaultModel
+        .manifestFetchError(NetworkError)
+
+    spec
+        .given(model)
+        .whenEvent(RetryClicked)
+        .then(assertThatNext(
+            hasModel(SelectCountryModel.FETCHING),
```

Comment:
This has to be a transition. (i.e.,) `SelectCountryModel.fetching()`. If it isn't, then we'll lose any information that was gathered between transitions.

Reply:
Right, I'll make this change.

Reply:
Done.